releaseNotes/SpringCloudAzureAppConfigurationConfig.md

```diff
@@ -14,6 +14,130 @@
 
 [Source code][source_code_web] | [Package (Maven)][package_web]
 
+## 5.9.0-beta.1 - January 11, 2024
+
+### Features Added
+
+* Added support for loading Feature Variants from Azure App Configuration.
```

Comment:
Can you add issue or PR links wherever possible?

Reply:
Did this where I could. Lots of the 4.7.0 features were added as a few PRs and we had no GitHub work items associated with them.